### PR TITLE
Add frontend rules for MSUnmerged.

### PR DESCRIPTION
**Pull request by todor-ivanov**

### CODE REVIEW

**Comment by todor-ivanov** [comment]
FYI: @amaltaro

File: frontend/backends-k8s-preprod.txt [hunk 1]
```diff
@@ -38,6 +38,7 @@
 ^/auth/complete/ms-monitor(?:/|$) ms-monitor.dmwm.svc.cluster.local
 ^/auth/complete/ms-output(?:/|$) ms-output.dmwm.svc.cluster.local
 ^/auth/complete/ms-rulecleaner(?:/|$) ms-rulecleaner.dmwm.svc.cluster.local
+^/auth/complete/ms-unmerged(?:/|$) ms-unmerged.dmwm.svc.cluster.local
```
Comment (amaltaro) [inline frontend/backends-k8s-preprod.txt:41]: @muhammadimranfarooqi can you please confirm whether this is correct?

**Review by amaltaro — APPROVED** [review]
It looks good from my side, thanks, Todor. However, it would be good if @muhammadimranfarooqi could review it as well.

**Comment by muhammadimranfarooqi** [comment]
@todor-ivanov 

Changes look fine.

**Comment by todor-ivanov** [comment]
Hi @muhammadimranfarooqi @vkuznet yet  another one to be merged too

**Comment by todor-ivanov** [comment]
FYI @amaltaro

**Pull request merged**